benchmarks/bun/bench.js

```diff
@@ -12,14 +12,15 @@ import { dirname } from 'path';
 import { fileURLToPath } from 'url';
 import systeminfo from "../systeminfo.js";
 const __dirname = dirname(fileURLToPath(import.meta.url));
-const resultsPath = '/var/results/bun';
+const resultsPath = '../results/bun';
 let frameworks = JSON.parse(readFileSync(__dirname+'/frameworks.json', 'utf-8'))
 
+const port = '3000';
```

Comment:
This is so the port can be referenced across the entire script, and passed then as env